07-volumes.md

```diff
@@ -23,22 +25,28 @@ volumes:
   db-data:
 ```
 
-An entry under the top-level `volumes` key can be empty, in which case it uses the platform's default configuration for
+The `db-data` volume is stored at `/var/lib/backup/data` and `/etc/data` container paths for backup and backend respectively.
+
+When `docker compose up` is run for the firs time, it creates the volume. Docker reuses the same volume when you run the command subsequently.
```

Comment:
typo:
```suggestion
When `docker compose up` is run for the first time, it creates the volume. Docker reuses the same volume when you run the command subsequently.
```

But also I think this paragraph could be more explicit:
```suggestion
`docker compose up` will create the volume if it doesn't already exist (such as on first execution). Otherwise, the existing volume is used (and will be recreated if it is manually deleted outside of Compose).
```